Branch-SDK/src/io/branch/referral/Branch.java

```diff
@@ -333,6 +346,33 @@ private Branch(Context context) {
 		linkCache_ = new HashMap<BranchLinkData, String>();
 	}
 
+
+	/**
+	 * <p>Singleton method to return the pre-initialised object of the type {@link Branch}.
+	 * Make sure your app is  instantiating {@link BranchApp} before calling this method</p>
```

Comment:
Sorry to be a stickler, but can you clean up the double spacing here and everywhere else? There's 2 spaces between *is* and *instantiating*